editors/arb-ltip/arb-ltip.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import { reducer, utils } from '../../document-models/arb-ltip-grantee';
+import Editor from './editor';
+import { createDocumentStory } from 'document-model-editors';
```

Comment:
after the rebase with `main` branch, this import should be changed by 'document-model-libs/utils'

Reply:
👍